web/app/main.js

```diff
@@ -35,8 +36,10 @@ var selectedCountryCode;
 var forceRemoteEndpoint = false;
 var customDate;
 var timelineEnabled = false;
+var reqLang = 'en';
```

Comment:
The translation is a mix of server (through i18n) and client, so far i had no luck to get the server understanding same locale as client but i ll have another try since it shoudl be the defaut behaviour of the i18n library. As matter of consistency i therefore also use the query string on client.